lighthouse-core/test/audits/redirects-test.js

```diff
@@ -9,41 +9,54 @@ const Audit = require('../../audits/redirects.js');
 const assert = require('assert');
 
 /* eslint-env mocha */
-const FAILING_REDIRECTS = {
+const FAILING_THREE_REDIRECTS = {
   startTime: 17,
+  url: 'http://exampel.com/',
   redirects: [
     {
-      endTime: 1,
-      responseReceivedTime: 5,
       startTime: 0,
       url: 'http://example.com/',
     },
     {
-      endTime: 16,
-      responseReceivedTime: 14,
       startTime: 11,
       url: 'https://example.com/',
     },
     {
-      endTime: 17,
-      responseReceivedTime: 15,
       startTime: 12,
       url: 'https://m.example.com/',
     },
   ],
 };
 
+const FAILING_TWO_REDIRECTS = {
+  startTime: 446.286,
+  url: 'http://lisairish.com/',
```

Comment:
perhaps better to just use example.com as an url so it's consistent with the rest?

Or is this a marketing stunt for your mom's website? 😆

Reply:
Well i tested this with these actual URLs, which have this set of redirects setup.